test/tst_boston_housing.jl

```diff
@@ -5,12 +5,12 @@ using MLDatasets
 
 
 @testset "Boston Housing" begin
-    X  = BostonHousing.features()
-    Y  = BostonHousing.targets()
+    X = BostonHousing.features()
+    Y = BostonHousing.targets()
     names = BostonHousing.feature_names()
     @test X isa Matrix{Float64}
     @test Y isa Matrix{Float64}
-    @test names == ["crim","zn","indus","chas","nox","rm","age","dis","rad","tax","ptratio","b","lstat"]
+    @test names == ["crim", "zn", "indus", "chas", "nox", "rm", "age", "dis", "rad", "tax", "ptratio", "b", "lstat"]
```

Comment:
These are unrelated changes.

Reply:
Yeah actually i added some space to the boston housing test file feature names, so just was reverting back to the original.

Reply:
Sure I will create a data folder and add the csv file there, I was just following the src code format of the boston housing market as it did quite similarly and I didn't want to write out of place code or change the sile strcuture of this library.

Reply:
If a new data folder outside the src folder has to be created just confirm me I will do the same, I was just following the boston housing dataset and its code structure.

Reply:
Cool if this gets merged, if any help in documentation is needed I am surely willing to help.

Reply:
> Let's add docs first and then we can make a new release.

Sure so how do we plan that? Should I follow the custom template , make the docs and send the PR to get it reviewed or would you like to handle it?